inspirehep/modules/records/serializers/json_literature.py

```diff
@@ -73,6 +72,12 @@ def _preprocess_result(result, original_record=None):
     """Add additional fields to output json"""
     record = result['metadata']
     ui_metadata = _get_ui_metadata(record)
+
+    if original_record is not None:
+        # If it is an db object then get citations from db
+        result['metadata']['citation_count'] = get_citations_count(
+            original_record
```

Comment:
Why you are adding it back? It's not required anywhere in details page and actually it slows loading of details page.